apps/emqx_message_transformation/src/emqx_message_transformation_schema.erl

```diff
@@ -108,8 +108,7 @@ fields(transformation) ->
                 hoconsc:array(ref(operation)),
                 #{
                     desc => ?DESC("operation"),
-                    required => true,
-                    validator => fun validate_operations/1
+                    required => true
```

Comment:
?
```suggestion
                    required => false
```

Reply:
and provided default value `[]`

Reply:
I've added the default value and omitted `required`.  That should be equivalent and enough, I believe?